bridges/snowbridge/pallets/system-v2/src/lib.rs

```diff
@@ -222,6 +246,37 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::call_index(5)]
+		#[pallet::weight(<T as pallet::Config>::WeightInfo::add_tip())]
+		pub fn add_tip(
+			origin: OriginFor<T>,
+			sender: AccountIdOf<T>,
+			message_id: MessageId,
+			amount: u128,
+		) -> DispatchResult {
+			T::FrontendOrigin::ensure_origin(origin)?;
+
+			let result = match message_id {
+				Inbound(nonce) => <T as pallet::Config>::InboundQueue::add_tip(nonce, amount),
+				Outbound(nonce) => <T as pallet::Config>::OutboundQueue::add_tip(nonce, amount),
+			};
+
+			if let Err(_) = result {
```

Comment:
I guess there's only one possible error, but I would still grab it and log it

Reply:
Added in https://github.com/paritytech/polkadot-sdk/pull/8271/commits/5bb4ffffe36527c6e2addb31fed55101e81609fc.